Source/UrlRequest_Windows.cpp

```diff
@@ -48,6 +48,20 @@ namespace UrlLib
             std::replace(path.begin(), path.end(), '/', '\\');
             return std::move(path);
         }
+
+        std::wstring ResolveSymlink(const std::wstring& path)
+        {
+            std::wstring resolvedPath{path};
+
+            while (std::filesystem::is_symlink(resolvedPath))
+            {
+                resolvedPath = std::filesystem::read_symlink(resolvedPath).wstring();
+            }
+
+            std::replace(resolvedPath.begin(), resolvedPath.end(), L'/', L'\\');
+
+            return std::move(resolvedPath);
+        }
```

Comment:
I just did a test in a standalone app and the WinRT APIs seem to follow symlinks just fine. Why is this necessary?

Reply:
Just tried in a Win32 BN playground. It looks like the WinRT Storage API is throwing these errors:
```
Exception thrown at 0x00007FF88C554C3C (KernelBase.dll) in Playground.exe: WinRT originate error - 0x80070005 : 'Cannot access the specified file or folder (D:\GitHub\BabylonJS\BabylonNative\Build\Win32\Apps\Playground\Debug\Scripts\experience2.js). Verify that the item is not marked with system or hidden file attributes.'.
Exception thrown at 0x00007FF88C554C3C (KernelBase.dll) in Playground.exe: WinRT originate error - 0x80070005 : 'Access is denied.'.
```
This is strange.

Reply:
I tested the UWP BN playground and it can load symlinks just fine, so something about using this API outside of an app container is causing an issue.

Reply:
Ok, I debugged the OS a bit. I think we should stop using WinRT APIs for loading local files for Win32. This code is way more complicated than we want it to be. We can use Win32 APIs directly instead. Just need to make sure to use the async versions (using the overlapped structure).